### PR TITLE
Daniel: Ex 15 & 16 done

### PR DESCRIPTION
**Pull request by dk1code**

### CODE REVIEW

**Comment by dk1code** [comment]
The task switches between asking for values in general and "the values". I find this a bit confusing.

File: 15-PB-conditionals-pt1/15-daniel.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+// 4.
+
+function functionFour(x) {
+  if (typeof (x) !== "string") {
+    return "The input is supposed to be a string!";
+  }
+  else {
+    x = "Py" + x;
+    if (x.includes("Py") === true) {
```
Comment (martl) [inline 15-PB-conditionals-pt1/15-daniel.js:65]: please use the startsWith() method

File: 15-PB-conditionals-pt1/15-daniel.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+console.log("\n4.");
+console.log("NaN:", functionFour(NaN)); // NaN: The input is supposed to be a string!
+console.log("ExampleString:", functionFour("ExampleString")); // ExampleString: ExampleString
+console.log("Empty string:", functionFour("")); // Empty string: 
+
+// 5.
+
+/* For this task I would have needed a two global vars - A hint that we are supposed to declared them globally would have been better. */
+
```
Comment (martl) [inline 15-PB-conditionals-pt1/15-daniel.js:79]: hahha :D don't overcomplicate things ;) nobody told you to use functions ;) If you do so, you have to fix your own problems

File: 15-PB-conditionals-pt1/15-daniel.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+console.log("\n5.");
+console.log("10, 20:", functionFive(10, 20)); //10, 20: 80
+console.log("10, 50:", functionFive(10, 50)); // 10, 50: 65
+console.log("40, 50:", functionFive(40, 50)); // 40, 50: 80
+
+// 6.
+
+function functionSix(x, y) {
+  if (x + y === 8 || Math.abs(x - y === 8) || x - y === -8) {
```
Comment (martl) [inline 15-PB-conditionals-pt1/15-daniel.js:97]: nice idea with Math.abs(), then you can get rid of the third check

File: 15-PB-conditionals-pt1/15-daniel.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+console.log("\n8.");
+console.log("0, 0:", functionEight(0, 0));
+console.log("7, 0:", functionEight(7, 0));
+console.log("0, 11:", functionEight(0, 11));
+
+// 9.
+
+function functionNine(x, y) {
+  if (x === y) {
+    return (x + y) * 3;
```
Comment (martl) [inline 15-PB-conditionals-pt1/15-daniel.js:154]: please create a variable for the sum

File: 15-PB-conditionals-pt1/15-daniel.js [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+console.log("\n9.");
+console.log("7, 7:", functionNine(7, 7)); // , 7: 42
+console.log("5, 7:", functionNine(5, 7)); // 5, 7: 12
+
+// 10.
+
+function functionTen(x) {
+  let y = 19;
+  if (Math.abs(x - y) > 19) {
```
Comment (martl) [inline 15-PB-conditionals-pt1/15-daniel.js:169]: please create a variable for the difference

File: 15-PB-conditionals-pt1/15-daniel.js (hunk 2 above)
Comment (dk1code) [reply 15-PB-conditionals-pt1/15-daniel.js:79]: True! :D This was just a remark, when I stumbled on that issue and didn't modified it according to my ongoing observations later. I went on going on as your remark stated.
I commented to the PR more appropriately,  that I find the instructions on the vars a bit inconsistent in this Ex.

File: 16-PB-conditionals-pt2/16-daniel.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+console.log(isBmiHigher);
+
+// 4.
+
+console.log(`4: Is Mark's BMI higher then John's: It is ${!isBmiHigher ? "not" : ""}.`);
+
+// 5.
+
+if (bmiMark > bmiJohn) {
+    console.log(`5: ${VALUES_MARK[0]}`)
```
Comment (Ungoot) [inline 16-PB-conditionals-pt2/16-daniel.js:35]: missing BMI in the log

File: 16-PB-conditionals-pt2/16-daniel.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+else {
+    console.log(`5: ${VALUES_JOHN[0]}`)
```
Comment (Ungoot) [inline 16-PB-conditionals-pt2/16-daniel.js:39]: missing BMI in the log

**Pull request closed without merging**